doc/CHANGELOG.md

```diff
@@ -8,33 +8,19 @@ Syntax: `- short text describing the change _(Your Name)_`
 
 ## 0.2.3 - UNRELEASED
 
-- Finish implementation of seed search _(Moritz)_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
-- _()_
```

Comment:
Please try to avoid such changes, as this introduces conflicts for other PRs. As we will release today anyway it is not a big problem this time though.